core/data/src/main/java/kr/co/cotton/core/data/news/datasource/LocalNewsDataSource.kt

```diff
@@ -0,0 +1,27 @@
+package kr.co.cotton.core.data.news.datasource
+
+import kotlinx.coroutines.flow.MutableStateFlow
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between